### DISCUSSION TITLE
how to disable the new selection GUI subtitles/audio track/chapters?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#15341] [Q&A] [answered]
Hi, there is a new feature, how can I diasble it and make it work like before?

Thank you :)

![subtitle](https://github.com/user-attachments/assets/2ae35469-72fe-495f-960e-0dbfea7d387a)

**Comment by llyyr** [comment, marked as answer]
https://github.com/mpv-player/mpv/blob/master/etc/restore-osc-bindings.conf

**Reply by geextahslex** [reply]
Thank you :)
